custom_components/omada/api/clients.py

```diff
@@ -25,6 +25,10 @@ class Clients(APIItems):
     def __init__(self, request):
         super().__init__(request, END_POINT, "mac", Client, data_key="data")
 
+    @property
+    def connected_clients(self) -> "list[Client]":
```

Comment:
This function seems redundant. You should be able to get the list of clients directly without creating a new list every call. APIItems passes forward __iter__ from the underlying dict and you can do the same with __len__ for the count.

Reply:
Agree